### PR TITLE
log training loss to wandb for train/val loss comparison

### PR DESCRIPTION
**Pull request by erinuclkwon**
Log training loss to wandb for training/validation comparison.

### CODE REVIEW

**Review by IFenton — APPROVED** [review]
LGTM. By default mine seems to plot training / validation on [two separate graphs](https://wandb.ai/turing-seaice/naive-unet-naive/runs/6j20nhoa). Did you want to try and get them to plot on the same graph by default? If not I'm happy for you to just merge this in now

**Comment by erinuclkwon** [comment]
Thanks for the confirmation. I will explore a bit more and see if I can get them to automatically plot on the same graph by default. If not, I will come back and merge.
